maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java

```diff
@@ -111,6 +111,7 @@ private void testing(File runFile) throws Exception {
 			String bndrun = getNamePart(runFile);
 			File bndrunCwd = new File(cwd, bndrun);
 			File bndrunResportsDir = new File(reportsDir, bndrun);
+			run.setProperty(Constants.DEFAULT_PROP_TARGET_DIR, cwd.getAbsolutePath());
```

Comment:
The correct thing to do is call `run.setBase(new File(cwd, getNamePart(runFile)))`, at the start of the try block. So any directories that Project.prepare might create are in that folder.